### DISCUSSION TITLE
Is there a way to switch the navbar back to before?

### DISCUSSION THREAD

**Discussion by nktnet1** [fuma-nama/fumadocs#982] [Q&A] [answered]
I like the old navbar design and have taken screenshots of it for documentation:

![image](https://github.com/user-attachments/assets/9ffcb35a-65c5-4794-856f-4ab2695e0129)

i.e. no border, and just a full-width underline beneath.

After updating to v14, it now looks like this:

![navbar-new](https://github.com/user-attachments/assets/b6e7c0d5-bf30-4ca1-b264-5fb1b14cc85a)

Is there a way to remove this border and bring back the full-width underline?

**Comment by fuma-nama** [comment, marked as answer]
You can see the version history and clone the previous navbar, go to home layout then replace default navbar via `nav.component` prop. All required contexts are exposed from `fumadocs-ui/provider`.
